test/test_helper.rb

```diff
@@ -11,6 +12,9 @@
 require 'minitest/pride'
 require 'factory_girl_rails'
 require 'support/factory_girl'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.